packages/discovery-provider/plugins/pedalboard/apps/relay/src/txRelay.ts

```diff
@@ -41,17 +37,17 @@ export const relayTransaction = async (
     const transaction = { nonce, gasLimit, to, value, data };
     await senderWallet.signTransaction(transaction);
 
-    logger.info({ requestId, senderWallet: address, address: userAddress, nonce }, "submitting transaction")
+    logger.info({ senderWallet: address, nonce }, "submitting transaction")
```

Comment:
do we still want sender wallet? or does the context having address serve as sufficient

Reply:
this will be important when we eventually move off from using generated wallets on acdc